Cargo.toml

```diff
@@ -30,7 +30,9 @@ fluvio             = { git = "https://github.com/infinyon/fluvio", branch = "mas
 fluvio-smartengine = { git = "https://github.com/infinyon/fluvio", branch = "master" }
 fluvio-protocol    = { git = "https://github.com/infinyon/fluvio", branch = "master" }
 fluvio-spu-schema  = { git = "https://github.com/infinyon/fluvio", branch = "master" }
+sasl2-sys          = { git = "https://github.com/infinyon/rust-sasl/", branch = "fix-cross-compiling" }
```

Comment:
can we fork instead of depend on branch?

Reply:
You mean have it point to `master` or a revision? Yeah, that should be fine.

Reply:
Could you create a PR in upstreaming repo to see if they can merge this change